Botany Plant Additions

FalseIncarnate opened the pull request:
Reintroduced a number of previously removed plants:
+ Tobacco
+ Space Tobacco
+ Tea Aspera
+ Tea Astra
+ Coffee Arabica
+ Coffee Robusta
+ Moonflower
+ Novaflower
+ Koibean

Fixed a typo preventing the mutation of Blood tomatoes into Killer
Tomatoes.
Added coffee arabica, tobacco, and tea aspera seeds to seed vending
machines.

Made Ice Tea inherit the traits of normal Tea, as Iced Coffee does this for normal Coffee (standardization)
Added Tea to recognized drinking glass contents, should no longer display as a "Glass of ...What?"

FalseIncarnate commented:
Closing, will put up a new one without the reagent and recipe changes.

FalseIncarnate commented:
Reopened PR with inclusion of new commit to revert the changes to reagents and recipes from the first commit. God it looks tacky.

Fox-McCloud commented on code/modules/reagents/Chemistry-Reagents.dm:
```diff
@@ -3131,47 +3131,46 @@ datum
+				description = "Coffee and ice, refreshing and cool."
+				color = "#102838" // rgb: 16, 40, 56
+				adj_temp = -5
 
```
All these indentation changes effectively eleminates these reagents from inheriting any traits of regular coffee, I believe. Please correct me if I'm wrong.

FalseIncarnate commented:
Fixed the indentation issue with coffee subtypes (iced coffee, soy latte, cafe latte), and added indentations to allow ice tea to inherit from tea.

Also fixed tea not being listed as a recognized drinking glass content, previously would result in a "Glass of ...What?"

Fox-McCloud commented:
Excellent; good catch on the iced tea; I missed that!
